README.md

```diff
@@ -362,6 +362,35 @@ galasactl runs download --name C1234 --destination /Users/me/my/folder
 A complete list of supported parameters for the `runs download` command is available [here](./docs/generated/galasactl_runs_download.md).
 
 
+## runs reset
+
+This command will reset a running test in the Ecosystem that is either stuck in a timeout condition or looping, by requeing the test.
+
+
+## Example
+
+The run "C1234" can be reset using the following command:
+
+```
+galasactl runs reset --name C1234
+```
+
+
+## runs cancel
+
+If after running `runs reset` the test is still not able to run through successfully, it can be abandoned with `runs cancel`.
+
+This command will cancel a running test in the Ecosystem . It will not delete any information that is already stored in the RAS about the test, it will only cancel the execution of the test.
+
```

Comment:
Add something like this ? : "Note: The cancel command does not wait for the server to complete the act of cancelling the test, but if the command succeeds, then the server has accepted the request to cancel the test."

Reply:
I have added this comment and similar to the reset command also